store/sqlstore/post_store.go

```diff
@@ -2339,6 +2339,31 @@ func (s *SqlPostStore) GetPostsByIds(postIds []string) ([]*model.Post, error) {
 	return posts, nil
 }
 
+func (s *SqlPostStore) GetEditHistoryForPost(postId string) ([]*model.Post, error) {
+	builder := s.getQueryBuilder().
+		Select("*").
+		From("Posts").
+		Where(sq.Eq{"Posts.OriginalId": postId}).
+		OrderBy("Posts.EditAt DESC")
+
+	queryString, args, err := builder.ToSql()
+	if err != nil {
+		return nil, errors.Wrap(err, "postsForExport_toSql")
```

Comment:
I think this(`postsForExport_toSql`) got wrongly copied. Maybe you can add something like this `failed to find post history` instead?